cmd/quadlet/main.go

```diff
@@ -358,4 +358,5 @@ func main() {
 func init() {
 	flag.BoolVar(&verboseFlag, "v", false, "Print debug information")
 	flag.BoolVar(&noKmsgFlag, "no-kmsg-log", false, "Don't log to kmsg")
+	flag.BoolVar(&isUser, "user", false, "Run as for systemd user")
```

Comment:
s/as for/as/

No idea why the GitHub UI doesn't show the edit button to propose changes.